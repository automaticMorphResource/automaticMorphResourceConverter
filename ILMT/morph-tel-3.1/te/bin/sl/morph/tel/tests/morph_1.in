1	kAnI	unk	
1	ikkada	unk
1	emuMxi	unk	
1	?	unk	
