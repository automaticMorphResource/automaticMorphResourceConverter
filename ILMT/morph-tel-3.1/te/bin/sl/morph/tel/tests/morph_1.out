1	kAnI	unk	<fs af='kAnI,avy,,,,,0,0_avy'>|<fs af='kAni,avy,,,,,V,V_avy'>|<fs af='kAni,avy,,,,,V,V_avy'>
2	ikkada	unk	<fs af='ikkada,n,,sg,,d,0,0'>|<fs af='ikkada,n,,sg,,o,ti,ti'>
3	emuMxi	unk	<fs af='emuMxi,avy,,,,,0,0_avy'>
4	?	unk	<fs af='&qus;,punc,,,,,,'>
